packages/perseus/src/widgets/__stories__/interactive-graph.stories.tsx

```diff
@@ -121,6 +122,19 @@ export const AllLockedLines = (args: StoryArgs): React.ReactElement => (
     />
 );
 
+export const AllLockedRays = (args: StoryArgs): React.ReactElement => (
+    <RendererWithDebugUI
+        apiOptions={{
+            flags: {
+                mafs: {
+                    segment: true,
+                },
+            },
```

Comment:
(talked offline)
Since this is in a different package, I'll hold off on doing that. If I keep adding to this file, I'll move the flags so they can be used in perseus though.